.github/workflows/test_sub.yaml

```diff
@@ -0,0 +1,60 @@
+---
+name: Test Sub
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - "*"
+jobs:
+  test:
+    name: Test
+    timeout-minutes: 30
+    strategy:
+      matrix:
+        os: ["ubuntu-latest"]
+        redis: ["5.0", "4.0", "3.2", "3.0"]
+        ruby: ["jruby-9.2.9.0", "2.3"]
+        driver: ["ruby"]
+    runs-on: ${{ matrix.os }}
+    env:
+      VERBOSE: true
+      TIMEOUT: 30
+      LOW_TIMEOUT: 0.03
+      DRIVER: ${{ matrix.driver }}
+      REDIS_BRANCH: ${{ matrix.redis }}
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Print environment variables
+        run: |
+          echo "TIMEOUT=${TIMEOUT}"
+          echo "LOW_TIMEOUT=${LOW_TIMEOUT}"
+          echo "DRIVER=${DRIVER}"
+          echo "REDIS_BRANCH=${REDIS_BRANCH}"
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
```

Comment:
There are two official actions for Ruby.

| repository | maintained by | supported versions |
| --- | --- | --- |
| https://github.com/actions/setup-ruby | GitHub | 2.4 - 2.7 |
| https://github.com/ruby/setup-ruby | Ruby community | 2.2 - 2.7, JRuby, TruffleRuby and Rubinius |